serverlessworkflow/modules/ROOT/pages/getting-started/create-your-first-workflow-service.adoc

```diff
@@ -85,49 +85,19 @@ Knative CLI::
 +
 --
 .Create a project using Knative CLI
-[source,shell]
+[source,shell,subs="attributes"]
 ----
-kn workflow create --name my-project --extension quarkus-jsonp,quarkus-smallrye-openapi
+kn workflow create \
+  --name serverless-workflow-hello-world \
+  --extension quarkus-jsonp,quarkus-smallrye-openapi \
+  --kogito-version={kogito_version} \
+  --quarkus-version={quarkus_version}
```

Comment:
```suggestion
  --kogito-version={kogito_version_redhat} \
  --quarkus-version={quarkus_version}
```

Also, add a conditional here to only render downstream.

Reply:
Done.